kytos/core/flow.py

```diff
@@ -225,9 +230,9 @@ def as_flow_mod(self, flow_type=FlowModCommand.OFPFC_ADD):
                 setattr(flow_mod, attribute_name, value)
 
         flow_mod.match = Match()
-        match_attributes = ['in_port', 'dl_src', 'dl_dst', 'dl_vlan',
-                            'dl_vlan_pcp', 'dl_type', 'nw_src', 'nw_dst',
-                            'tp_dst']
+        match_attributes = ['wildcards', 'in_port', 'dl_src', 'dl_dst',
```

Comment:
Because wildcards are automatically managed by pyof, we don't need to have ``wildcards`` here. Please, remove it to minimize the changes.